playwright-e2e/lib/component/Question.ts

```diff
@@ -124,6 +127,13 @@ export default class Question {
     return this.toLocator().locator('ddp-date');
   }
 
+  async checkAndFillInInput(value: string, opts: { inputText?: string } = {}): Promise<void> {
+    const { inputText } = opts;
+    const checkbox = new Checkbox(this.page, { label: value, root: this.toLocator() });
+    await checkbox.check();
+    await checkbox.fill(inputText);
+  }
```

Comment:
Some UI checkboxes opens up an input field for user to fill in. Purpose of this function is to enable test author to write lesser code.